includes/definitions/discovery/gigabyte-bmc.yaml

```diff
@@ -0,0 +1,40 @@
+mib: GIGABYTE-BMC
+modules:
+    os:
+        hardware: GIGABYTE-BMC::fruPartNumber.0
+        serial: SNMPv2-MIB::snmpSetSerialNo.0
+    processors:
+        data:
+            -
+                oid: processorCollection
+                num_oid: '.1.3.6.1.4.1.15370.552.2.1.1'
+
+    sensors:
+        voltage:
+            data:
+                -
+                    oid: sensorCollection
+                    num_oid: '.1.3.6.1.4.1.15370.554.1.1.4.{{ $index }}'
+                    descr: Voltage
+                    divisor: 10
+#
```

Comment:
The collection of all the sensors (voltage, CPU utilisation, temperatures, fanspeeds etc.) are all found at .554.1.1.4.
The voltages are for example at:
```
['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.23', 'sensor_name' => 'P_12V'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.24', 'sensor_name' => 'P_5V'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.25', 'sensor_name' => 'P_3V3'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.26', 'sensor_name' => 'P_5V_STBY'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.27', 'sensor_name' => 'P_1V_AUX_PCH'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.28', 'sensor_name' => 'P_VBAT'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.29', 'sensor_name' => 'P_VCCIN_CPU0'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.30', 'sensor_name' => 'P_VDDQ_AB'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.31', 'sensor_name' => 'P_VDDQ_CD'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.32', 'sensor_name' => 'P_VCCSA'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.33', 'sensor_name' => 'VR_P0_VIN'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.34', 'sensor_name' => 'VR_P0_VCCSA'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.35', 'sensor_name' => 'VR_P0_VCCIO'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.36', 'sensor_name' => 'VR_P0_VCCIN'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.37', 'sensor_name' => 'VR_DIMMG0_VIN'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.39', 'sensor_name' => 'VR_DIMMG1_VIN'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.43', 'sensor_name' => 'P_12V_CPU0'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.44', 'sensor_name' => 'P_12V_GPU1'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.45', 'sensor_name' => 'P_12V_GPU2'],
        ['sensor_oid' => '.1.3.6.1.4.1.15370.554.1.1.4.46', 'sensor_name' => 'P_12V_MB'],
```
The following is an example of other OID values:
![image](https://user-images.githubusercontent.com/24249616/95212631-2f2cad00-07ee-11eb-8b61-901e60e2aa30.png)

Any idea how I can split this up into Voltages, Temperatures, Fanspeeds etc. when {{ index }} is different?

Reply:
You will need to use the skip_values option to match based on the sensorName or index, name is more likely to be generically correct.